### PR TITLE
[14.0] [MIG] account_statement_import_paypal: Migration to v14.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT29944

please @pedrobaeza @joao-p-marques review this

### CODE REVIEW

**Review by joao-p-marques — COMMENTED** [review]
Code review :+1: 

Travis is red @CarlosRoca13

**Comment by joao-p-marques** [comment]
@CarlosRoca13 Travis is still red, although I don't see any error or test failure...

**Review by ernestotejeda — COMMENTED** [review]
code review :+1:

**Review by pedrobaeza — COMMENTED** [review]
You need to add migration scripts for renaming the model/table.

**Comment by CarlosRoca13** [comment]
@pedrobaeza done

File: account_statement_import_paypal/migrations/14.0.1.0.0/pre-migrate.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelid import openupgrade
```
Comment (pedrobaeza) [inline account_statement_import_paypal/migrations/14.0.1.0.0/pre-migrate.py]: s/openupgradelid/openupgradelib

**Comment by CarlosRoca13** [comment]
Oups! Changed

**Pull request merged**